app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -357,6 +363,72 @@ def test_update_experiment_probe_sets_error(self):
             },
         )
 
+    def test_update_experiment_outcomes(self):
+        user_email = "user@example.com"
+        experiment = NimbusExperimentFactory.create(
+            status=NimbusExperiment.Status.DRAFT,
+            application=NimbusExperiment.Application.DESKTOP,
+            primary_outcomes=[],
+            secondary_outcomes=[],
+        )
+        outcomes = [
+            o.slug for o in Outcomes.by_application(NimbusExperiment.Application.DESKTOP)
+        ]
+        primary_outcomes = outcomes[: NimbusExperiment.MAX_PRIMARY_PROBE_SETS]
```

Comment:
Ah, this is the reason for `ignore = E203` in flake8? Not used to seeing a space in this notation (probably thanks to flake8 defaults), but it does make it clearer with that long constant name

Reply:
Yeah I think black is doing it and since you can't configure black it's just one of those things where black is like 'this is how I do it, deal with it' so I had to add the exception 🤷‍♀️